src/engine.cpp

```diff
@@ -234,18 +236,21 @@ void Engine::load_networks() {
         networks_.small.load(binaryDirectory, options["EvalFileSmall"]);
     });
     threads.clear();
+    threads.ensure_network_replicated();
```

Comment:
this is also unnecessary right cause the modify_and_replicate i think calls the load already or is that lazy?

Reply:
Subsequent calls are basically no-ops, so I'd be inclined to leave these calls as is. Especially since you're not even sure a glance if it's correct to remove them.